contracts/proto/faas/v1/faas.proto

```diff
@@ -71,6 +71,8 @@ message ApiWorker {
 
 message SubscriptionWorker {
   string topic = 1;
+  // optional dead-letter queue where events should be sent if they can't be delivered to/processed by this worker.
+  string dead_letter_queue = 2;
```

Comment:
changing `dead_letter_queue` -> `dead_letter` here might be better wording for supporting different sink types for deadletter messages.

Reply:
done